client/tests/simulator_tests.rs

```diff
@@ -51,9 +52,23 @@ fn get_started_tpm() -> (TpmSim, TcpTpm) {
 }
 
 #[test]
-fn test_get_manufacturer_id() {
+fn test_get_capability_manufacturer_id() {
     let (_sim_lifeline, mut tpm) = get_started_tpm();
-    assert!(get_manufacturer_id(&mut tpm).is_ok());
+
+    const CMD: GetCapabilityCmd = GetCapabilityCmd {
```

Comment:
is there a reason to not do the normal

```rust
let cmd =  GetCapabilityCmd {
....
```

This is fairly standard, and usually `const` is reservered for global constants

Reply:
Fixed